ochre/Equipment/WaterHeater.py

```diff
@@ -413,8 +413,12 @@ class HeatPumpWaterHeater(ElectricResistanceWaterHeater):
     def __init__(self, hp_only_mode=False, water_nodes=12, **kwargs):
         super().__init__(water_nodes=water_nodes, **kwargs)
 
+        self.low_power_hpwh = kwargs.get('Low Power HPWH', False)
```

Comment:
I think it'd be better to move these parameters into the hpxml.py file. I can
start to work on that.

Reply:
Agreed, when we have HPXML support for 120V HPWHs, we'll get rid of this and a few other things that are just for testing purposes.